internal/project/runner.go

```diff
@@ -25,6 +28,7 @@ func (r *Runner) Run(ctx context.Context, path string) (*Process, error) {
 	cmd := exec.CommandContext(runCtx, path)
 	cmd.Stdout = output
 	cmd.Stderr = output
+	cmd.Args = append(cmd.Args, r.args...)
```

Comment:
What about passing the arguments as third argument to `exec.CommandContext` ?

Reply:
Totally makes sense. I was not paying attention that this is possible.